src/Bindings/LuaState.cpp

```diff
@@ -868,7 +868,7 @@ cLuaState::cStackValue cLuaState::WalkToValue(const AString & a_Name)
 		// Remove the previous value from the stack (keep only the new one):
 		lua_remove(m_LuaState, -2);
 	}  // for elem - path[]
-	return std::move(cStackValue(*this));
+	return cStackValue(*this);
```

Comment:
The compiler was complaining though. See first comment for error: https://news.ycombinator.com/item?id=9915300

```
    foo.cpp:44:25: error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]
        auto result = std::move(worker.run());

    foo.cpp:44:25: note: remove std::move call here
        auto result = std::move(worker.run());
```